ocaml/testsuite/tests/typing-layouts-missing-cmi/function_arg.ml

```diff
@@ -10,8 +10,6 @@ module = "function_b.ml"
 script = "rm -f function_a.cmi"
 ***** expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
```
[%%expect{|
Line 1, characters 28-56:
1 | let f2 : Function_b.fun_t = fun ~arg1:_ ~arg2 () -> arg2
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error: Function arguments and returns must be representable.
       The layout of Function_a.t is any, because
         the .cmi file for Function_a.t is missing.
       But the layout of Function_a.t must be a sublayout of '_representable_layout_3, because
         it's used as a function argument.
       No .cmi file found containing Function_a.t.
       Hint: Adding "function_a" to your dependencies might help.
|}]
```

In this error message and others, the combination of the location and error message is confusing.
  * The location identifies the function as a whole
  * The error message zeroes in on `Function_a.t`, as it's the type of one of the arguments to the function.

The underlined location is the ambient referent. If the main subject of the error message *isn't* the ambient referent, then I think there should be a sentence that identifies the subject.

In this case, I'd like something like this as the second sentence of the error message:

```
This function has an argument of type Function_a.t.
```

Reply:
(The same comment goes for `f2` and onward. `f0` and `f1` are fine as the referent is the underlined piece.)

Reply:
filed ticket